wee_slack.py

```diff
@@ -3778,6 +3778,8 @@ def unhtmlescape(text):
 
 
 def unwrap_attachments(message_json, text_before):
+    if 'original_url' in message_json and not config.link_previews:
```

Comment:
`original_url` is a field in the attachment itself, not in the message, so you would need to move it into the for loop below.

Reply:
Whoops, fixed.